brewdo

```diff
@@ -331,14 +354,18 @@ end
 
 # switch ownership/groups recursively on HOMEBREW_HOME, HOMEBREW_CACHE
 def switch_owners(new_user, new_home_group, new_cache_group)
-  home_uid, home_gid = file_owner(HOMEBREW_HOME + '/.git') 
-  cache_uid, cache_gid = file_owner(HOMEBREW_CACHE + '/Formula')
+  home_uid, home_gid = file_owner(HOMEBREW_HOME + '/Homebrew')
```

Comment:
Is this right? `/usr/local/Homebrew`? I am not using `/usr/local` anymore—my `$HOMEBREW_HOME` is set to `$HOME/Library/Homebrew`—but there's no `Homebrew` directory there.

Reply:
Well I had to change it from `.git`, because there's no longer a `.git` in there... Homebrew itself is moved into the `/usr/local/Homebrew` dir for me. The last modification date is yesterday, so it's clearly getting used on my system.